space-framework/entry.sh

```diff
@@ -23,8 +23,37 @@ function echoerr() {
 }
 
 function run_space_manager() {
-    echo "--< Starting space-manager >--"
-    if ! space_manager -v=${VERBOSITY} ; then
+    echo "--< Starting space-manager 1 >--"
+    KUBECONFIG=
+    kubectl config set-cluster space-mgt --server="https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}" --certificate-authority=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    kubectl config set-credentials space-mgt --token="$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)"
+    kubectl config set-context space-mgt --cluster=space-mgt --user=space-mgt 
+    kubectl config use-context space-mgt
+    KUBECONFIG=/home/spacecore/.kube/config
+
+    echo "Apply space manager CRDs."
+    kubectl apply -f /home/spacecore/config/crds
+
+    echo "Waiting for kcp to be ready... this may take a while."
```

Comment:
Why are we waiting here ? Maybe we should create the providers in some function and call it?

Reply:
I moved the code to a separate function.